test/common/assertSnapshot.js

```diff
@@ -24,8 +33,20 @@ function replaceWindowsPaths(str) {
   return common.isWindows ? str.replaceAll(path.win32.sep, path.posix.sep) : str;
 }
 
-function replaceFullPaths(str) {
-  return str.replaceAll('\\\'', "'").replaceAll(path.resolve(__dirname, '../..'), '');
+function replaceWindowsDriveLetter(str) {
+  if (!common.isWindows) {
+    return str;
+  }
+  const currentDriveLetter = path.parse(process.cwd()).root.substring(0, 1).toLowerCase();
+  const regex = new RegExp(`${currentDriveLetter}:`, 'gi');
+  return str.replaceAll('\\\'', "'").replaceAll(regex, '');
```

Comment:
Wouldn't that fail if `process.cwd()` is on a different drive than the test file?

Reply:
`make test` will set the current working directory to node source dir, right?

Reply:
There are other ways to run the test that using `make test`, but in any case, we shouldn't use CWD when `__dirname` gives a more reliable info

Reply:
Ah, you are right. I get your point. Updated, thanks!

Reply:
```suggestion
  const currentDriveLetter = __dirname.charAt(0).toLowerCase();
  const regex = new RegExp(`${currentDriveLetter}:`, 'gi');
  return str.replaceAll('\\\'', "'").replaceAll(regex, '');
```

Reply:
Thanks, I managed to remove the need of this function.